### PR TITLE
Do we need a name to match election.yaml?

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Or we do not need it at all?

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]

**Comment by kalkayan** [comment]
The name is also not used in the application, we can remove it if not used in the gitops part.

File: elections/2020_SC/voters.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: 2020 Steering Committee Election
+name: 2020 Kubernetes Steering Committee Election
 details: Eligibility is determined by Kubernetes contributions for the year.  See the main 2020
```
Comment (SergeyKanzhelev) [inline elections/2020_SC/voters.yaml:2]:
```suggestion
details: Eligibility is determined by Kubernetes contributions for the year.  See the main 2020
```

**Comment by SergeyKanzhelev** [comment]
Let's see what @markyjackson-taulia and @jberkus think about removing the name. I'd go with the minimal required number of fields to minimize the copy/paste needed to schedule the election

**Comment by markyjackson-taulia** [comment]
+1 to minimal fields. I think this can be removed

**Comment by jberkus** [comment]
I actually wanted the name to be a display name.  If we're not using it currently, then that's a bug.

**Comment by jberkus** [comment]
+1

That was from when I thought this file would be human -readable

**Pull request merged**